docs/references/experimental/path-dependency.md

```diff
@@ -0,0 +1,57 @@
+# Path Dependency
+
+## What is path dependency?
+
+Path dependency is a reliance on the file system as the source, as opposed to a package manager registry or URL. A path dependency may or may not have transitive dependencies.
+
+For example, in the following `go.mod` file, with `gomod` analysis and the `--experimental-analyze-path-dependencies` flag, `fossa-cli` would consider `../vendor/squirrel` a path dependency in the final dependency graph. If path dependency analysis is disabled, `fossa-cli` would ignore this dependency completely and only show transitive dependencies originating from `../vendor/squirrel`. In such a case, license and copyright obligations originating from `../vendor/squirrel` will not be captured in FOSSA's findings, and subsequent software bill of materials generated.
+
+```go
+// Example go.mod file
+module tester
+
+go 1.14
+
+replace github.com/Masterminds/squirrel => ../vendor/squirrel
+require github.com/Masterminds/squirrel v1.4.0
+```
+
+In the FOSSA UI, path dependencies are shown with the `Local` tag.
+
+## How are path dependencies scanned?
+
+Path dependencies are scanned using the "CLI license scan," similar to how [vendor dependencies](./../../features/vendored-dependencies.md) are scanned by default. A "CLI license scan" performs a license scan at the path and uploads the results of these scans to the provided FOSSA endpoint.
+
+For performance reasons, the FOSSA service caches the results of a path dependency by the combination of its `(project id, path, hash of path's content)`. Due to this caching setup, it is normal for the first analysis to take some time, especially for larger projects, but future analyses of dependencies with the same information should be fast.
+
+In the event that caching is causing problems, FOSSA can be made to rescan this kind of dependency by:
+- Running `fossa analyze` with the `--force-vendored-dependency-rescans` flag, or
```

Comment:
We decided to reuse `--force-vendored-dependency-rescans` flag after discussing with scott initially, but happy to change this, if we prefer to.